solutions/fully-configurable/variables.tf

```diff
@@ -70,7 +72,7 @@ variable "event_notifications_access_tags" {
 
 variable "service_credential_names" {
   type        = map(string)
-  description = "The mapping of names and roles for service credentials that you want to create for the Event Notifications instance. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-event-notifications/tree/main/solutions/fully-configurable/DA-types.md#service-credential-secrets"
+  description = "The mapping of names and roles for service credentials that you want to create for the Event Notifications instance. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-event-notifications/tree/main/solutions/fully-configurable/DA-types.md#service-credential-secrets)."
```

Comment:
Attached link is not correct.

Regarding the description, can we improve? I am not sure how to make it more clear, but as a new user I had to read the docs to understand.